### DISCUSSION TITLE
hi, how to plot different(e.g. training acc and val acc) in tensorboard in same window?

### DISCUSSION THREAD

**Discussion by JimXu1989** [Lightning-AI/pytorch-lightning#5833] [General]
Hi, thanks for your great work!

I want plot training accuracy and validation accuracy in same tensorboard, so

I can realize overfiting very convient

Thanks a lot!!

**Comment by SkafteNicki** [comment]
you can achieve this by calling the `add_scalars` method of the underlying summarywriter:
``` python
    def training_step(self, batch, batch_idx):
        x, y = batch
        y_hat = self(x)
        loss = F.cross_entropy(y_hat, y)
        # dummy example
        self.logger.experiment.add_scalars('losses', {'train': loss, 'val': loss+torch.randn_like(loss)},
                                           global_step = self.global_step)
        return loss
```
this will create two curves in the same window:
![Capture10](https://user-images.githubusercontent.com/24896311/106874575-03570080-66d6-11eb-8ea8-39fdff346dd0.PNG)
however, do note that it will mess with your runs (left side of the image) because it is essentially adding a new run/tensorboard file per scalar.
